cudax/cmake/cudaxBuildCompilerTargets.cmake

```diff
@@ -11,6 +11,12 @@
 #   be linked into the developer build targets, as they include both
 #   cudax.compiler_interface and cccl.compiler_interface_cppXX.
 
+find_package(libcudacxx ${cudax_VERSION} EXACT CONFIG
+  NO_DEFAULT_PATH # Only check the explicit path in HINTS:
+  HINTS "${CCCL_SOURCE_DIR}/lib/cmake/libcudacxx/"
+)
+thrust_create_target(cudax.libcudacxx)
```

Comment:
Don't use `thrust_create_target` here -- just use the `libcudacxx::libcudacxx` that is found by `find_package(libcudacxx ...)`.

```suggestion
```